### ISSUE TITLE
Plugins not loading on Neptune client start

### ISSUE THREAD

**Issue by Inrixia** [uwu/neptune#45]
As noted in https://github.com/Inrixia/neptune-plugins/issues/106

When the client is started no plugins load even if enabled until they are toggled or refreshed.

**Comment by Veriiix** [comment]
Any news? Maybe implement another check after launch to reload the plugins if not applied

**Comment by jacobrreed** [comment]
Yup, seeing same behavior. I have to go into neptune settings and manually toggle off and on each plugin to get them to load.

**Comment by omstr** [comment]
> Yup, seeing same behavior. I have to go into neptune settings and manually toggle off and on each plugin to get them to load.

Really? is it just a select few plugins supposed to be bugged? I only have 2 and Volume Scroll doesn't load, but I've not had to reload Discord RPC since enabling it
